parts/linux/cloud-init/artifacts/labels.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Update Labels for Kubernetes nodes
+
+set -euo pipefail
+
+for kubelet_label in $(echo $KUBELET_NODE_LABELS | sed "s/,/ /g")
+do
+  kubectl label --kubeconfig /var/lib/kubelet/kubeconfig --overwrite nodes $HOSTNAME $kubelet_label
```

Comment:
Do we need to check if the node has joined the cluster?

Reply:
The labels.service executes after the kubelet.service has finished starting up. Is that enough to ensure this, or do you think that more should be done? 
https://github.com/Azure/AgentBaker/blob/603788b85aa4502f05fcd2544d6d9dd89e47b150/parts/linux/cloud-init/artifacts/labels.service#L3

I don't believe the label-nodes.service does anything extra from my understanding, but that doesn't mean its right

Reply:
this code runs as a systemd service so it re-runs on failure. I think it's ok if kubelet is not ready for a short period of time. it will eventually be working.